entrypoints/content/pages/settings/SafetyForm.tsx

```diff
@@ -0,0 +1,142 @@
+
+// Components:
+import { Switch } from '../../components/ui/switch'
+import { Slider } from '../../components/ui/slider'
+import { RadioGroup, RadioGroupItem } from '../../components/ui/radio-group'
+import { Label } from '../../components/ui/label'
+
+// Functions:
+const SafetyForm = () => {
+  // Constants:
+  const {
+    // TODO: Disable all inputs across /settings when isUserPreferencesLoading is true.
+    isLoading: isUserPreferencesLoading,
+    loadUserPreferences,
+    safety,
+    setWebsiteWarningEnabled,
+    setWebsiteWarningWarnAt,
+    setWebsiteWarningPosition,
+  } = useUserPreferences()
+
+  // State:
+  const [bannerPositionPreference, setBannerPositionPreference] =
+    useState<WebsiteFlagBannerPosition>(WebsiteFlagBannerPosition.Bottom)
+
+  // Effects:
+  useEffect(() => {
+    loadUserPreferences()
+  }, [])
+
+  // Return:
+  return (
+    <>
+      <div className='flex flex-col'>
+        <h3 className='text-base font-medium'>Website Warning</h3>
+        <p className='text-xs font-regular text-brand-secondary'>
+          Control how safety banners are shown on unsafe websites.
+        </p>
+      </div>
+      <div className='flex flex-row justify-between h-fit align-middle items-center'>
+        <h3 className='text-sm font-regular'>Display Safety Banner</h3>
+        {/* TODO: Verify Implementation */}
+        <Switch
+          className='scale-75'
+          disabled={!isUserPreferencesLoading}
+          onCheckedChange={WebsiteWarning =>
+            setWebsiteWarningEnabled(WebsiteWarning)
+          }
+        />
+      </div>
+      <div className='flex flex-col gap-1'>
+        <h3 className='text-sm font-regular'>Intelligent Warning</h3>
+        <p className='text-xs font-regular text-brand-secondary pb-1'>
+          Not all websites are equally dangerous. Choose when we should warn
+          you, based on the website’s risk level.
+        </p>
+        {/* TODO: use setWebsiteWarningWarnAt */}
+        <Slider
+          step={25}
+          thumbClassName='h-4 w-4 cursor-pointer hover:bg-border-primary transition-all'
+          disabled={!isUserPreferencesLoading}
+        />
+        <div className='flex flex-row justify-between pt-1'>
+          <span className='text-xs font-regular text-brand-secondary'>
+            Minimal
+          </span>
+          <span className='text-xs font-regular text-brand-secondary'>
+            Moderate
+          </span>
+          <span className='text-xs font-regular text-brand-secondary'>
+            Severe
+          </span>
+        </div>
+      </div>
+      <div className='flex flex-col gap-1'>
+        <h3 className='text-sm font-regular'>Safety Banner Position</h3>
+        <p className='text-xs font-regular text-brand-secondary pb-3'>
+          Select where the safety banner should be shown.
+        </p>
+        <div className='flex flex-row justify-start gap-10 w-2/3 min-h-24'>
+          {/* TODO: use setWebsiteWarningPosition */}
```

Comment:
i'd say we go ahead and kick out `setBannerPositionPreference` and use `setWebsiteWarningPosition` only